fn main() -> Result<(), Box<dyn std::error::Error>> {
    aoc2023::run()
}
